.github/workflows/ci.yml

```diff
@@ -11,9 +11,6 @@ jobs:
     - uses: workarea-commerce/ci/eslint@v1
       with:
         args: '**/*.js'
-    - uses: workarea-commerce/ci/stylelint@v1
```

Comment:
I assume this is being removed because there's no CSS in this plugin, but is that really necessary?

Reply:
Yeah, `stylelint` exits with status code of `1` if it doesn't lint anything.